### DISCUSSION TITLE
Manejo de Inputs CODE TREE P2 T2

### DISCUSSION THREAD

**Discussion by cgandarillas2** [IIC2133-PUC/2024-2#166] [Tarea 2] [answered]
Hola, no he modificado nada en el código y la lectura del input que nos dieron no me recoge todo el codigo del tree:

Para este input: 98184923 3 0 0 0

Me imprime:

ID: 98184923, CODE_TREE: 3
ID: 0, CODE_TREE: 0
Error reading CODE_TREE

Mi duda es si es el comportamiento esperado y debo manejarlo así o no?

edit: ya cache xd

**Comment by 4n1t4** [comment, marked as answer]
Debes modificar la lectura inicial recuerda se agregó el parámetro del largo del code tree , para que puedan manejar de mejor manera su tarea. Debes modificar lo necesario.
